### PR TITLE
Update faros-airbyte-cdk version

### PR DESCRIPTION
**Pull request by Roma-Kyrnis**
## Description

> Provide description here

## Type of change
- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

**Review by cjwooo — APPROVED** [review]

File: destinations/faros-destination/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "faros-destination",
-  "version": "0.1.36",
+  "version": "0.1.44",
```
Comment (Roma-Kyrnis) [inline destinations/faros-destination/package.json:3]: Is it ok to use 0.1.44 version? Because other connectors has 0.1.36 version?

**Review by cjwooo — APPROVED** [review]

**Comment by Roma-Kyrnis** [comment]
Before merge I'll update package-lock.json

File: destinations/faros-destination/package.json (hunk 1 above)
Comment (cjwooo) [reply destinations/faros-destination/package.json:3]: Yea we bumped up the destination version separately, but I think we might bump everything up to 0.1.45 next just to get everything consistent.

Comment (Roma-Kyrnis) [reply destinations/faros-destination/package.json:3]: Ok

**Pull request merged**